DllImportGenerator/DllImportGenerator/TypePositionInfo.cs

```diff
@@ -72,9 +72,9 @@ public static TypePositionInfo CreateForType(ITypeSymbol type, IEnumerable<Attri
         }
 
 #nullable enable
```

Comment:
Can we remove this now?

Reply:
I still haven't enabled nullability in the DllImportGenerator project, just in Ancillary.Interop. I'll open an issue to track enabling it and enable it in here in the next PR.